TemplateApplication.xcodeproj/xcshareddata/xcschemes/TemplateApplication.xcscheme

```diff
@@ -79,7 +79,7 @@
       <CommandLineArguments>
          <CommandLineArgument
             argument = "--disableFirebase"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
Would be great if we can reset the flag to NO here. On the other hand, it might be the most convenient way to run the app. What do you think what should be the default? If this is the default, we should probably add a sentence to the README and modify it slightly when we talk about Firebase.